Makefile

```diff
@@ -54,12 +55,24 @@ gx: $(gx_bin) $(gx-go_bin)
 deps: gx
 	$(gx_bin) install --global
 	$(gx-go_bin) rewrite
+
 test: deps
 	go test -tags silent -v ./...
+test_sharness: sharness_deps
+	@sh sharness/run-sharness-tests.sh
+
+sharness_deps: clean_sharness 
```

Comment:
like with gx, there is a clever way to make this so that makefile won't re-download sharness if its there.

```
$(sharness):
  steps to download sharness
```
The value of `sharness` should probably be: `sharness/test/sharness`. See my comments on the install file. You can replicate the gx rules pretty much.